_posts/2019-02-27-stripe-connect-part-1.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: "Implementing Stripe Connect in Rails | Part 1"
+date: 2019-02-27 16:00:00
+categories: ["rails", "stripe"]
+author: "luciano"
+---
+
+In a recent project for [OmbuLabs](https://www.ombulabs.com), we were looking for a payment solution that allows users of our platform to easily start accepting online payments but at the same time we wanted to take a fee from it.
```

Comment:
"online payments WHILE at the same ALLOWING US TO COLLECT a fee from each TRANSACTION THEY MADE".